### DISCUSSION TITLE
Migrate in production - drift after adding data to db

### DISCUSSION THREAD

**Discussion by yazer79** [prisma/prisma#7538] [Q&A] [answered]
w10 - nodejs - mysql - prisma CLI - v2.23 (now upgraded to 2.24.1)

I have a production database running locally for development and i had prisma fine and applied some migrations before
After adding a new model to the schema and using CLI trying to apply it with "npx prisma migrate dev", I got a error "DB drift detected"

Does adding data to the DB cause a drift in between migrations? or there's another reason?

**Comment by janpio** [comment, marked as answer]
No, it should not do that. Can you please open an issue about this? We hope to have much better communication about _why_ drift is being detected soon, but your specific use case might be interesting enough to double check.

**Reply by yazer79** [reply]
I can't post my db in public. Is that ok?

**Reply by yazer79** [reply]
Do I need to remove the relations I added to the schema?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
